game-service/src/main/java/org/libertybikes/game/core/GameRound.java

```diff
@@ -483,20 +483,11 @@ public void startGame() {
         // Issue a countdown to all of the clients
         gameState = State.STARTING;
 
-        sendToClients(clients.keySet(), new OutboundMessage.StartingCountdown(STARTING_COUNTDOWN));
-        delay(TimeUnit.SECONDS.toMillis(STARTING_COUNTDOWN));
-
-        paused.set(false);
-        for (Player p : getPlayers())
-            if (STATUS.Connected == p.getStatus())
-                p.setStatus(STATUS.Alive);
-        broadcastPlayerList();
-        if (!gameRunning.get()) {
-            ManagedScheduledExecutorService exec = executor();
-            if (exec != null)
-                exec.submit(this);
+        ManagedScheduledExecutorService exec = executor();
+        if (exec != null) {
```

Comment:
The `executor()` method should never return null (unless something goes horribly wrong, in which case we can't cope anyway), so this null check isn't necessary

Reply:
We do the null check in all the other spots we call executor() so I was trying to be consistent, should I remove all the null checks?